yang/ietf-l3vpn-ntw.yang

```diff
@@ -753,38 +753,34 @@ module ietf-l3vpn-ntw {
                   description
                     "An identifier of an active VPN instance profile.";
                 }
+                leaf bearer-reference {
+                    if-feature "vpn-common:bearer-reference";
+                    type string;
+                    description
+                      "This is an internal reference for the service
+                      provider.";
+                  }
                 uses vpn-common:service-status;
                 container connection {
                   description
                     "Encapsulation types.";
-                  leaf encapsulation-type {
-                    type identityref {
-                      base vpn-common:encapsulation-type;
+                  leaf connection-type {
+                    type enumeration {
+                        enum LOCAL {
+                          description
+                            "evpn bgp-instance is operationally up";
```

Comment:
Why are we talking about EVPN/operationally?

I'm not sure to get the intent here.

Reply:
Sorry I have copy/paste from other model. My mistake